ui/article-aem-renderer/src/main/java/org/cru/godtools/article/aem/db/ArticleRepository.kt

```diff
@@ -1,14 +1,16 @@
 package org.cru.godtools.article.aem.db
 
+import androidx.annotation.AnyThread
 import androidx.annotation.WorkerThread
 import androidx.room.Dao
 import androidx.room.Transaction
 import org.cru.godtools.article.aem.model.Article
 
 @Dao
 abstract class ArticleRepository internal constructor(private val db: ArticleRoomDatabase) {
+    @AnyThread
```

Comment:
the `@MainThread`, `@UiThread`, `@WorkerThread`, and `@AnyThread` annotations help drive some automated lint checks to ensure you are calling methods from the correct threads.

Quite a few of these were present in the java code and ported over when I converted to Kotlin, in the process I cleaned up some methods and made it so they could be called from AnyThread.

In reality my feeling is that all `suspend` methods should be safe to call from `@AnyThread` because if it is going to do a blocking operation, it should use `withContext()` to offload that operation to either `Dispatchers.Default` or `Dispatchers.IO`. But seeing as coroutines is still relatively new for us, we haven't had good discipline around that coding pattern yet.